packages/core/src/constants.ts

```diff
@@ -6,3 +6,22 @@ export const CONFIG = {
   CONTRACT_OPTIONS: 'CONTRACT_OPTIONS',
   EVENT_WATCHER_OPTIONS: 'EVENT_WATCHER_OPTIONS',
 }
+
+export const DB_PREFIXES = {
+  TRANSACTIONS: '00',
+  BLOCK_HEADERS: '01',
+  PREDICATES: '02',
+  DEPOSITS: '03',
+  EXITS: '04',
+  ACCOUNTS: '05',
+  KEYSTORES: '06',
+  EXITABLE_ENDS: '07',
+  FINALIZED_EXITS: '08',
+  EXITED_RANGES: '09',
+  FAILED_TRANSACTION_IMPORTS: '10',
+  SEEN_EVENTS: '11',
+  LATEST_BLOCK: '12',
+  LATEST_STATE: '13',
+  LAST_LOGGED_BLOCK: '14',
+  LAST_SYNCED_BLOCK: '15',
+}
```

Comment:
These should probably be a single byte value to avoid weird encoding / decoding unknown behavior. So these DBs store things as bytes so if we use strings to `put` then we may have some weird thing where one key is encoded `12ab` and the other key `12ab1b`, and that would be bad!

So probably best to use something like:
```
SEEN_EVENTS: Buffer.from([255]),
```
That way we know it's a single byte and we know there won't be some unintended overlap.

Reply:
Oh another thing you can do is you can create an object which allows you to "register" prefixes, and it does the Buffer management on the backend. I'd say that looking at this will be helpful: 

https://github.com/bcoin-org/bdb